recipes/lat_lon_parser/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "lat_lon_parser" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cd43cfa833a82981ca3306af7aeb6fa40b3431b7391b82710a2443244cadbf81
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

The Python version constraint should match the one in `run`
```suggestion
    - python >=2.7
```